src/test/java/com/powsybl/network/conversion/server/NetworkConversionTest.java

```diff
@@ -108,6 +106,136 @@ public void setup() {
+
+        HvdcLine hvdcLine = network.getHvdcLine("FRA1AA_BBE1AA_hvdcline");
+        equipmentInfos = networkConversionService.toEquipmentInfos(hvdcLine, networkUuid, VariantManagerConstants.INITIAL_VARIANT_ID);
+        expectedEquipmentInfos = EquipmentInfos.builder()
+                .networkUuid(networkUuid)
+                .variantId(VariantManagerConstants.INITIAL_VARIANT_ID)
+                .id("FRA1AA_BBE1AA_hvdcline")
+                .name("FRA1AA_BBE1AA_hvdcline")
+                .type(IdentifiableType.HVDC_LINE.name())
+                .voltageLevels(Set.of(VoltageLevelInfos.builder().id("FRA1AA1").name("FRA1AA1").build(), VoltageLevelInfos.builder().id("BBE1AA5").name("BBE1AA5").build()))
+                .substations(Set.of(SubstationInfos.builder().id("FRA1AA").name("FRA1AA").build(), SubstationInfos.builder().id("BBE1AA").name("BBE1AA").build()))
+                .build();
+        assertEquals(expectedEquipmentInfos, equipmentInfos);
+
+        TwoWindingsTransformer twoWindingsTransformer = network.getTwoWindingsTransformer("BBE1AA2  BBE3AA1  2");
+        equipmentInfos = networkConversionService.toEquipmentInfos(twoWindingsTransformer, networkUuid, VariantManagerConstants.INITIAL_VARIANT_ID);
+        expectedEquipmentInfos = EquipmentInfos.builder()
+                .networkUuid(networkUuid)
+                .variantId(VariantManagerConstants.INITIAL_VARIANT_ID)
+                .id("BBE1AA2  BBE3AA1  2")
+                .name("BBE1AA2  BBE3AA1  2")
+                .type(IdentifiableType.TWO_WINDINGS_TRANSFORMER.name())
+                .voltageLevels(Set.of(VoltageLevelInfos.builder().id("BBE1AA2").name("BBE1AA2").build(), VoltageLevelInfos.builder().id("BBE1AA1").name("BBE1AA1").build()))
+                .substations(Set.of(SubstationInfos.builder().id("BBE1AA").name("BBE1AA").build()))
+                .build();
+        assertEquals(expectedEquipmentInfos, equipmentInfos);
+
+        ThreeWindingsTransformer threeWindingsTransformer = network.getThreeWindingsTransformer("BBE1AA_w3t");
+        equipmentInfos = networkConversionService.toEquipmentInfos(threeWindingsTransformer, networkUuid, VariantManagerConstants.INITIAL_VARIANT_ID);
+        expectedEquipmentInfos = EquipmentInfos.builder()
+                .networkUuid(networkUuid)
+                .variantId(VariantManagerConstants.INITIAL_VARIANT_ID)
+                .id("BBE1AA_w3t")
+                .name("BBE1AA_w3t")
+                .type(IdentifiableType.THREE_WINDINGS_TRANSFORMER.name())
+                .voltageLevels(Set.of(VoltageLevelInfos.builder().id("BBE1TR1").name("BBE1TR1").build(), VoltageLevelInfos.builder().id("BBE1TR2").name("BBE1TR2").build(), VoltageLevelInfos.builder().id("BBE1TR3").name("BBE1TR3").build()))
+                .substations(Set.of(SubstationInfos.builder().id("BBE1AA").name("BBE1AA").build()))
+                .build();
+        assertEquals(expectedEquipmentInfos, equipmentInfos);
+    }
```

Comment:
Add also a test for a line present in testCase.xiidm (for example 'BBE1AA1  BBE2AA1  1')

Reply:
done